parsing/src/processor.ts

```diff
@@ -93,8 +93,27 @@ function getItems(data: any[]): PartDataInterface {
         .filter((entry: any) => entry.Classes)
         .flatMap((entry: any) => entry.Classes)
         .forEach((entry: any) => {
+            // There are two exception products we need to check for and add to the parts list
+            if (entry.ClassName === "Desc_NuclearWaste_C") {
+                // Note that this part id is NuclearWaste, not Uranium Waste
+                parts["NuclearWaste"] = { 
+                    name: "Uranium Waste",
+                    stackSize: 500, //SS_HUGE
+                    isFluid: false,
+                    isFicsmas: false,
+                };
+            }
+            if (entry.ClassName === "Desc_PlutoniumWaste_C") {
+                parts["PlutoniumWaste"] = {
+                    name: "Plutonium Waste",
+                    stackSize: 500, //SS_HUGE
+                    isFluid: false,
+                    isFicsmas: false,
+                };
+            }
+
             // Ensures it's a recipe, we only care about items that are produced within a recipe.
-            if (!entry.mProducedIn) return;
+            if (!entry.mProducedIn) return;                
```

Comment:
Wonky spacing

Reply:
We should consider adding a linter to break the build when this stuff is added.

Reply:
We could yes